common/build/eslint-config-fluid/package.json

```diff
@@ -42,19 +45,19 @@
 		"eslint-plugin-import-x": "~4.16.1",
 		"eslint-plugin-jsdoc": "~55.0.2",
 		"eslint-plugin-promise": "~7.2.1",
-		"eslint-plugin-react": "~7.37.5",
-		"eslint-plugin-react-hooks": "~5.2.0",
+		"eslint-plugin-react": "~7.33.2",
+		"eslint-plugin-react-hooks": "~4.6.0",
 		"eslint-plugin-tsdoc": "~0.4.0",
 		"eslint-plugin-unicorn": "~48.0.1",
 		"eslint-plugin-unused-imports": "~4.2.0"
 	},
 	"devDependencies": {
-		"@fluid-tools/markdown-magic": "file:../../../tools/markdown-magic",
```

Comment:
Couldn't get both the eslint-config and client pipelines to run correctly with this dep, so removing it temporarily.